task_6/map-practice.js

```diff
@@ -0,0 +1,9 @@
+function mapPractice(niz) {
+  const novNiz = niz.map(function(item, index, array) {
+    const a = item - index;
+    return a;
+  });
+  return novNiz;
+}
+
```

Comment:
gledaj da kada prosledjujes anonimne funkcije kao callback one uvek budu arrow funkcije jer tako ne moras da pazis na `this` bindovanje i kod je dosta citljiviji.
I ne moras da navodis treci, cak ni drugi argument ako ih ne koristis tako da ti `array`argument ne treba
```suggestion
return niz.map((item, index) => item - index);
```